pvsite_datamodel/sqlmodels.py

```diff
@@ -61,9 +61,8 @@ class SiteSQL(Base, CreatedMixin):
     )
     latitude = sa.Column(sa.Float)
     longitude = sa.Column(sa.Float)
-    capacity_kw = sa.Column(
-        sa.Float, comment="The physical limit on the production capacity of the site"
-    )
+    inverter_capacity_kw = sa.Column(sa.Float, comment="The inverter capacity of the site")
+    module_capacity_kw = sa.Column(sa.Float, comment="The PV module nameplate capacity of the site")
```

Comment:
Yep, sounds good—I went ahead and put `capacity_kw` back in, and I'm populating `inverter_capacity_kw` with the values from the `capacity_kw` column.

Reply:
thanks, can you set the default to equal `capacity_kw` column?

Reply:
Yep—since postgres doesn't support default values based on another column, I added a trigger function that sets `inverter_capacity_kw` to `capacity_kw` if `inverter_capacity_kw` is null